### ISSUE TITLE
No indicator on startup

### ISSUE THREAD

**Issue by heathbar** [vikoadi/indicator-kdeconnect#1]
Running 'indicator-kdeconnect' gives the output below and I do not see the indicator in the panel. I'm running elementary freya; kdeconnect v0.5.2.1.

*\* (indicator-kdeconnect:11118): WARNING *_: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-YvR7cbHkn9: Connection refused
*_ Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
*\* Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
QDBusConnection: session D-Bus connection created before QCoreApplication. Application may misbehave.
QDBusConnection: session D-Bus connection created before QCoreApplication. Application may misbehave.

(kdeconnectd:11121): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(kdeconnectd:11121): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(kdeconnectd:11121): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(kdeconnectd:11121): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(kdeconnectd:11121): Gtk-WARNING *_: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
kdeconnect(11121)/kdeconnect_kded Daemon::Daemon: Starting KdeConnect daemon
*_ Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service

**Comment by vikoadi** [comment]
A notification icon will only show up after KDE Connect application started in connected android.

**Comment by shyamsathyanathan** [comment]
I have the same issue and even though the application on android is open. The android app is not detecting the KDE Connect on my laptop even though both of them are on the same network. I even tried connecting by tethering my laptop to my phone.

My error is different however:  

```
** (indicator-kdeconnect:6313): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-EUYyrGK5mk: Connection refused
```
